PolyEngine/RenderingDevice/OpenGL/Src/Proxy/GLShaderProgram.cpp

```diff
@@ -9,14 +9,19 @@ UNSILENCE_MSVC_WARNING()
 
 using namespace Poly;
 
+const String GLShaderProgram::SHADERS_INCLUDE_DIR = "Shaders/";
```

Comment:
Same deal as with `WHITESPACES`, make it static, remove declaration from header.